client/app/main/school-reports/school-reports.controller.js

```diff
@@ -3,47 +3,61 @@
 var app = angular.module('app');
 
 function SchoolReports($scope, Auth, uiGridGroupingConstants,
-	Student, AbsenceRecord) {
-	$scope.chronicAbsentGridOptions = {
-		enableSorting: true,
+  Student, AbsenceRecord) {
+  $scope.chronicAbsentGridOptions = {
+    enableSorting: true,
     enableGridMenu: true,
     enableFiltering: true,
     treeRowHeaderAlwaysVisible: false,
     exporterPdfDefaultStyle: {fontSize: 9},
     exporterPdfTableHeaderStyle: {fontSize: 10, bold: true, color: 'grey'},
-    exporterPdfHeader: { text: 'Chronically Absent Students', style: 'headerStyle' },
+    exporterPdfHeader: {
+      text: 'Chronically Absent Students',
+      style: 'headerStyle'
+    },
     exporterPdfOrientation: 'landscape',
     exporterPdfPageSize: 'LETTER',
     exporterPdfMaxGridWidth: 500,
-    exporterPdfFooter: function ( currentPage, pageCount ) {
-      return { text: currentPage.toString() + ' of ' + pageCount.toString(), style: 'footerStyle' };
+    exporterPdfFooter: function(currentPage, pageCount) {
+      return {
+        text: currentPage.toString() + ' of ' + pageCount.toString(),
+        style: 'footerStyle'
+      };
     },
```

Comment:
I was thinking, maybe we should add the date of the report to the footer or header?